testing/ncids-css-testing/stories/components/nci-card/nci-card.scenarios.js

```diff
@@ -106,48 +91,64 @@ module.exports = [
 				width: 1024,
 				height: 768,
 			},
+		],
+	},
+	{
+		label: 'nci-card test-case card-group without description',
+		storyId: 'components-card-test-cases--card-group-without-description',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 360,
+				height: 360,
+			},
 			{
-				label: 'widescreen',
-				width: 1400,
+				label: 'tablet',
+				width: 640,
+				height: 360,
+			},
+			{
+				label: 'desktop',
+				width: 1024,
 				height: 768,
 			},
 		],
 	},
 	/* PostInteractWait Accounts for Image Zoom Transition */
 	{
-		label: 'nci-card hover',
-		storyId: 'components-nci-card--default',
+		label: 'nci-card default hover',
+		storyId: 'components-card-default--default',
 		hoverSelector: '.nci-card a',
 		postInteractionWait: 500,
```

Comment:
can you change `postInteractionWait` to `readySelector` on all of these that have it. An arbitrary number will slow down the tests